Header header
float64 latitude
float64 longitude
float64 elevation
float64 latStaDev
float64 lonStaDev
float64 eleStaDev